src/lib/DriveProvider.spec.jsx

```
@@ -0,0 +1,91 @@
+
+jest.mock('cozy-sharing', () => {
+  const SharingProvider = jest.fn(({ children }) => children)
+  SharingProvider.displayName = 'SharingProvider'
+  return {
+    __esModule: true,
+    default: SharingProvider,
+    NativeFileSharingProvider: ({ children }) => children
+  }
+})
+
+jest.mock('cozy-ui/transpiled/react/providers/Alert', () => ({ children }) => children)
```

ESLint

[error] Replace `'cozy-ui/transpiled/react/providers/Alert',·()·=>·({·children·})·=>·children` with `⏎··'cozy-ui/transpiled/react/providers/Alert',⏎··()·=>⏎····({·children·})·=>⏎······children⏎`

(prettier/prettier)